src/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -33,19 +33,25 @@
 using Microsoft.Identity.Client.Cache.Items;
 using Microsoft.Identity.Client.OAuth2;
 using Microsoft.Identity.Client.TelemetryCore;
+using Microsoft.Identity.Client.Internal.Broker;
 
 namespace Microsoft.Identity.Client.Internal.Requests
 {
     internal class SilentRequest : RequestBase
     {
         private readonly AcquireTokenSilentParameters _silentParameters;
+        private AuthenticationRequestParameters _authenticationRequestParameters;
+        BrokerFactory brokerFactory = new BrokerFactory();
+        private IBroker Broker;
+
         public SilentRequest(
             IServiceBundle serviceBundle,
             AuthenticationRequestParameters authenticationRequestParameters,
             AcquireTokenSilentParameters silentParameters)
             : base(serviceBundle, authenticationRequestParameters, silentParameters)
         {
             _silentParameters = silentParameters;
+            _authenticationRequestParameters = authenticationRequestParameters;
```

Comment:
>_authenticationRequestParameters [](start = 12, length = 32)

similar q's as those for the interactive request. I believe this already exists on the base object